### PR TITLE
reshape #6643

### LINKED ISSUE #6643: reshape

### PR DESCRIPTION
**Pull request by eswhy28**
Closes #6643

### CODE REVIEW

**Comment by guillesanbri** [comment]
Hi! Thanks a lot for contributing to Ivy. A few things to review:
- There are changes to various files related to the IDE, project configuration, etc. this should be removed from the PR (.idea directories).
- Seems like the formatting does not correspond to the one followed by Ivy, this can be checked with the [pre-commit hook](https://lets-unify.ai/ivy/contributing/0_setting_up.html#pre-commit). Also, I guess your auto formatting has changed the indent of the `swapaxes` test.

**Comment by eswhy28** [comment]
I used pre-commit it passed the both checks flake8 and Black. I have removed .idea from PR and push. I hope i did what you asked correctly.
Thank You

**Comment by eswhy28** [comment]
> Hi! Thanks a lot for contributing to Ivy. A few things to review:
> 
> * There are changes to various files related to the IDE, project configuration, etc. this should be removed from the PR (.idea directories).
> * Seems like the formatting does not correspond to the one followed by Ivy, this can be checked with the [pre-commit hook](https://lets-unify.ai/ivy/contributing/0_setting_up.html#pre-commit). Also, I guess your auto formatting has changed the indent of the `swapaxes` test which should be undone as well. slightly_smiling_face

Please Kindly check. Thank You

**Comment by eswhy28** [comment]
I don't know why the pull request fails when creating hypothesis directory

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_transpose_like_operations.py [hunk 1]
```diff
@@ -82,13 +82,14 @@ def st_dtype_arr_and_axes(draw):
     ),
     native_array=helpers.array_bools(num_arrays=1),
 )
-def test_numpy_swapaxes(
+def test_numpy_swapaxpes(
```
Comment (guillesanbri) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_transpose_like_operations.py:85]: This is not correctly written

**Comment by guillesanbri** [comment]
You don't have to delete the idea folder as this has some files used project-wide, it should be the same as before the commit, i.e. no files should be modified other than the one with the new function and the new test. You can check the right-most window called `files changed`, now it says that you have modified ~80 files, and instead it should be only two. Don't hesitate to ask if anything is unclear :slightly_smiling_face:

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_transpose_like_operations.py (hunk 1 above)
Comment (eswhy28) [reply ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_transpose_like_operations.py]: i have fixed this issue

**Comment by guillesanbri** [comment]
Hi, there are still some changes on top of yours, this is partly due to the force-push of master, which brought some commits, but there are also deletions that have been commited that would override other existing commits if merged. This should be fixed :slightly_smiling_face:

**Comment by eswhy28** [comment]
Hi!, i hope the recent commit solved what you asked

**Comment by guillesanbri** [comment]
hi, unfortunately, there are still changes in the PR that would override changes in the main repo, you can check the changes I am referring to in the `Files changed` tab, which reflect the changes of your PR

**Comment by eswhy28** [comment]
hi can you kindly check please

**Comment by guillesanbri** [comment]
Looks good to me now! We are currently looking into something related to this PR internally, but I will get back to you asap! Thanks once again!

**Comment by eswhy28** [comment]
Okay thank you...

**Comment by guillesanbri** [comment]
Closing this as the function was already implemented in [another file](). However, the function and test implementation is correct and would have been merged otherwise.

**Pull request closed without merging**